### DISCUSSION TITLE
Stuck on Preparing Task

### DISCUSSION THREAD

**Discussion by CatsOverDogsFightMe** [lllyasviel/Fooocus#3315] [Q&A]
I did all the steps and read the readme and troubleshooting page and everything but it’s still always just stuck on preparing task when I try to generate an image. I can’t find anyone else with this issue? Pls help!
![IMG_9580](https://github.com/user-attachments/assets/8357594b-db9e-498c-8306-184a83ed4239)

**Comment by mashb1t** [comment]
Please provide more information:
- system specs incl. GPU specs
- swap allocation
- performance information of task manager or similar
- Fooocus terminal logs
